ios/Podfile

```diff
@@ -58,7 +58,7 @@ target 'BetterSocial' do
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
   # use_flipper!
-  use_flipper!({ 'Flipper-Folly' => '2.6.7', 'Flipper-RSocket' => '1.4.3' , 'Flipper' => '0.88.0' })
+  # use_flipper!({ 'Flipper-Folly' => '2.6.7', 'Flipper-RSocket' => '1.4.3' , 'Flipper' => '0.88.0' })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The specific versions for Flipper-Folly, Flipper-RSocket, and Flipper have been removed. This could lead to potential compatibility issues if the latest versions of these libraries are not compatible with your project. If you want to use the latest versions, it's recommended to test thoroughly to ensure there are no breaking changes. If you want to stick with the known working versions, you should keep the specific versions.
```diff
-   # use_flipper!({ 'Flipper-Folly' => '2.6.7', 'Flipper-RSocket' => '1.4.3' , 'Flipper' => '0.88.0' })
+   use_flipper!({ 'Flipper-Folly' => '2.6.7', 'Flipper-RSocket' => '1.4.3' , 'Flipper' => '0.88.0' })
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->